docs/site/tutorials/introducing_x10.ipynb

```diff
@@ -0,0 +1,550 @@
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "tRIJp_4m_Afz",
+        "colab": {}
+      },
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\"); { display-mode: \"form\" }\n",
+        "// Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "// you may not use this file except in compliance with the License.\n",
+        "// You may obtain a copy of the License at\n",
+        "//\n",
+        "// https://www.apache.org/licenses/LICENSE-2.0\n",
+        "//\n",
+        "// Unless required by applicable law or agreed to in writing, software\n",
+        "// distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "// See the License for the specific language governing permissions and\n",
+        "// limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "sI1ZtrdiA4aY"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        " <td>\n",
+        "  <a target=\"_blank\" href=\"https://www.tensorflow.org/swift/tutorials/introducing_x10\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+        " </td>\n",
+        " <td>\n",
+        "  <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/swift/blob/master/docs/site/tutorials/introducing_x10.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        " </td>\n",
+        " <td>\n",
+        "  <a target=\"_blank\" href=\"https://github.com/tensorflow/swift/blob/master/docs/site/tutorials/introducing_x10.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        " </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "8sa42_NblqRE"
+      },
+      "source": [
+        "# Introducing X10\n",
+        "\n",
+        "By default, Swift For TensorFlow performs tensor operations using eager dispatch. This allows for rapid iteration, but isn't the most performant option for training machine learning models.\n",
+        "\n",
+        "The newly available [X10 tensor library](https://github.com/tensorflow/swift-apis/blob/master/Sources/x10/swift_bindings/doc/API_GUIDE.md) adds a high-performance backend to Swift for TensorFlow, leveraging tensor tracing and the [XLA compiler](https://www.tensorflow.org/xla). This tutorial will introduce X10 and guide you through the process of updating a training loop to run on GPUs or TPUs."
```

Comment:
Perhaps remove "newly available" to have this tutorial age better?

Reply:
Good point, that dated this. Removed.